wrench/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MPL-2.0"
 
 [dependencies]
 base64 = "0.3"
-bincode = "1.0.0-alpha2"
+bincode = "0.8"
```

Comment:
I don't know anything about bincode's versioning, but 0.8 is the only version supporting serde 1.0, that's why I chose it